07/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Shapes
 
-Finally! We have been building skills for this moment! You have learned most of the GLSL foundations, types and functions. You have practiced your shaping equations over and over. Now is the time to put it all together. You are up for this challenge! In this chapter you'll learn how to draw simple shapes in a parallel procedural way.
+Finally! We have been building skills for this moment! You have learned most of the GLSL foundations, types and functions. You have practiced your shaping equations over and over. Now is the time to put it all together. Are you up for this challenge! In this chapter you'll learn how to draw simple shapes in a parallel procedural way.
```

Comment:
This wasn't a question; but an encouraging statement.

Reply:
It's good to encourage instead of questioning : )